app/utils/fields-validations.ts

```diff
@@ -118,63 +122,63 @@ function dateValidation(obj: any, errors): boolean | ValidationErrors {
 }
 
 function appellantNamesValidation(obj: object) {
-  const schema = Joi.object({
-    givenNames: Joi.string().required().messages({ 'string.empty': i18n.validationErrors.givenNames }),
-    familyName: Joi.string().required().messages({ 'string.empty': i18n.validationErrors.familyName })
+  const schema = joi.object({
+    givenNames: joi.string().required().messages({ 'string.empty': i18n.validationErrors.givenNames }),
+    familyName: joi.string().required().messages({ 'string.empty': i18n.validationErrors.familyName })
 
   }).unknown();
   return validate(obj, schema);
 }
 
 function contactDetailsValidation(obj: object) {
-  const schema = Joi.object({
-    'selections': Joi.alternatives()
+  const schema = joi.object({
+    'selections': joi.alternatives()
       .try(
-        Joi.array()
+        joi.array()
           .items(
-            Joi.string().valid('email', 'text-message')
+            joi.string().valid('email', 'text-message')
           ),
-        Joi.string().valid('email', 'text-message'))
+        joi.string().valid('email', 'text-message'))
       .required()
       .messages({
         'any.required': i18n.validationErrors.contactDetails.selectOneOption
       }),
-    'email-value': Joi.when('selections', {
-      is: Joi.alternatives()
-        .try(Joi.array().items(Joi.string().valid('email'), Joi.string().valid('text-message')),
-          Joi.string().valid('email')).required(),
-      then: Joi.string()
+    'email-value': joi.when('selections', {
+      is: joi.alternatives()
+        .try(joi.array().items(joi.string().valid('email'), joi.string().valid('text-message')),
+          joi.string().valid('email')).required(),
+      then: joi.string()
         .optional()
         .email({ minDomainSegments: 2, tlds: { allow: [ 'com', 'net', 'co.uk' ] } })
         .messages({
           'string.empty': i18n.validationErrors.emailEmpty,
           'string.email': i18n.validationErrors.emailFormat
         }),
-      otherwise: Joi.strip()
+      otherwise: joi.strip()
     }),
-    'text-message-value': Joi.when('selections', {
-      is: Joi.alternatives()
+    'text-message-value': joi.when('selections', {
+      is: joi.alternatives()
         .try(
-          Joi.array().items(Joi.string().valid('email'), Joi.string().valid('text-message')),
-          Joi.string().valid('text-message')).required(),
-      then: Joi.string()
+          joi.array().items(joi.string().valid('email'), joi.string().valid('text-message')),
+          joi.string().valid('text-message')).required(),
+      then: joi.mobilePhoneNumber().format('e164')
```

Comment:
so I would probably do...
Joi.extend(MobilePhoneNumberExtension).format.....

Reply:
Thank Toni, I will make the suggested change on the next commit